ui/k-info-panel/switch_info.kytos

```diff
@@ -48,6 +53,8 @@
    props: ["content"],
    data () {
      return {
+       chosen_action: '',
+       chosen_interface: '',
```

Comment:
Could you rename this variable to reflect the fact that it can hold multiple interfaces, something like `chosen_interfaces`, what do you tink? For future readers that will maintain it'll facilitate to understand what it holds.

Reply:
Yes, it makes more sense. That name was due to me not knowing I had to load a set of interfaces to the api instead of just only one. Changed on commit [0c3aaac67309e7495cd3d6ba58ef0e6b121d6368](https://github.com/kytos-ng/topology/pull/81/commits/0c3aaac67309e7495cd3d6ba58ef0e6b121d6368)